python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -152,6 +152,8 @@ def get$Name(self):
         ("varianceCol", "column name for the biased sample variance of prediction.",
          None, "TypeConverters.toString"),
         ("aggregationDepth", "suggested depth for treeAggregate (>= 2).", "2",
+         "TypeConverters.toInt"),
+        ("parallelism", "number of threads to use when fitting models in parallel (>= 1).", "1",
```

Comment:
I think the description should be more general.  Is the plan to put the shared param in here or in OneVsRest first?

Reply:
No worry, #19110 will merge first and then I will merge it to this PR.